### PR TITLE
fixing paths

### PR DESCRIPTION
**Pull request by mstafamajid**

### CODE REVIEW

**Comment by mstafamajid** [comment]
fixing paths

File: 2023-2024/week-2/Group A - 8-puzzle depth FS/app.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import random
 import tkinter
-from board import EightPuzzleGUI  # Make sure you have the 'EightPuzzleGUI' class defined in 'board.py'
+from board import EightPuzzleGUI 
+from os import path# Make sure you have the 'EightPuzzleGUI' class defined in 'board.py'
+
+FILE_PATH=path.abspath("2023-2024\week-2\Group A - 8-puzzle depth FS\\solution.txt")
```
Comment (kodo-yousif) [inline 2023-2024/week-2/Group A - 8-puzzle depth FS/app.py:6]: this is still static path 

if i change directory name nothing will work 

please use this way 

image

File: 2023-2024/week-2/Group A - 8-puzzle depth FS/board.py [hunk 2]
```diff
@@ -11,12 +12,11 @@ def __init__(self, root, solution_path):
...
 
         # Load images for numbers 1 to 8 and blank
-        self.images = [
-            PhotoImage(file=f"2023-2024/week-2/puzzle_board/assets/{i}.png")
+        self.images = [ PhotoImage(file= path.abspath(f"2023-2024/week-2/puzzle_board/assets/{i}.png"))
             for i in range(1, 9)
         ]
         self.images.append(
-            PhotoImage(file="2023-2024\week-2\Group A - 8-puzzle depth FS\\0.png")
+            PhotoImage(file=path.abspath("2023-2024\week-2\Group A - 8-puzzle depth FS\\0.png"))
```
Comment (kodo-yousif) [inline 2023-2024/week-2/Group A - 8-puzzle depth FS/board.py:19]: same as `solution.txt`

File: 2023-2024/week-2/Group A - 8-puzzle depth FS/board.py [hunk 3]
```diff
@@ -11,12 +12,11 @@ def __init__(self, root, solution_path):
         self.current_step = 0
 
         # Load images for numbers 1 to 8 and blank
-        self.images = [
-            PhotoImage(file=f"2023-2024/week-2/puzzle_board/assets/{i}.png")
+        self.images = [ PhotoImage(file= path.abspath(f"2023-2024/week-2/puzzle_board/assets/{i}.png"))
```
Comment (kodo-yousif) [inline 2023-2024/week-2/Group A - 8-puzzle depth FS/board.py:15]: same as `solution.txt`

**Review by kodo-yousif — APPROVED** [review]
you added absolute path and it should work on my PC now however

it is not the correct way and it's not stable for the future

please resolve my requests

**Review by kodo-yousif — CHANGES_REQUESTED** [review]
you added absolute path and it should work on my PC now however

it is not the correct way and it's not stable for the future

please resolve my requests

**Review by kodo-yousif — APPROVED** [review]

**Pull request merged**